packages/orchestrator/internal/sandbox/template/template.go

```diff
@@ -3,13 +3,14 @@ package template
 import (
 	"errors"
 
+	"github.com/e2b-dev/infra/packages/orchestrator/internal/sandbox/block"
 	"github.com/e2b-dev/infra/packages/shared/pkg/storage"
 )
 
 type Template interface {
 	Files() *storage.TemplateCacheFiles
-	Memfile() (*Storage, error)
-	Rootfs() (*Storage, error)
+	Memfile() (block.ReadonlyDevice, error)
+	Rootfs() (block.ReadonlyDevice, error)
 	Snapfile() (File, error)
 	Close() error
 }
```

Comment:
pls fix `closeTemplate` so it returns if `t.Snapfile()` if error, the error correctly